### PR TITLE
feat: support bmm converter in dynamo

### PR DESCRIPTION
**Pull request by bowang007**
# Description

Support bmm converter for dynamo

- New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by narendasan** [comment]
@bowang007 lint the PR

File: tests/py/dynamo/converters/test_bmm.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            def forward(self, input, mat2):
+                return torch.bmm(input, mat2)
+            
+        inputs = [torch.randn(*input_shape), torch.randn(*mat2_shape)]
+
+
+        self.run_test(
+            BMM(), 
+            inputs,
+            expected_ops={},
```
Comment (gs-olive) [inline tests/py/dynamo/converters/test_bmm.py:28]: disable_passes=True

Comment (gs-olive) [reply tests/py/dynamo/converters/test_bmm.py:28]: Consider changing default arg in `run_test`

**Comment by gs-olive** [comment]
Consider switching to True default:
https://github.com/pytorch/TensorRT/blob/a65c95cbd4496034bd4413aca060db7c24618fbd/tests/py/dynamo/converters/harness.py#L219

File: py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py [hunk 2]
```diff
@@ -315,6 +315,17 @@ def aten_ops_matmul(
         args[1],
     )
 
+@dynamo_tensorrt_converter(torch.ops.aten.bmm.default)
```
Comment (gs-olive) [inline py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:318]: Put decorator in above function + rebase

Comment (bowang007) [reply py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py]: thanks!

File: tests/py/dynamo/converters/test_bmm.py (hunk 1 above)
Comment (bowang007) [reply tests/py/dynamo/converters/test_bmm.py]: used `disable_passes=True`

**Review by gs-olive — APPROVED** [review]
Looks good to me, pending CI validation for the added test case!

**Comment by bowang007** [comment]
> Looks good to me, pending CI validation for the added test case!

Is CI fixed now?

**Comment by gs-olive** [comment]
It should be working, but I see there is some sort of Torch version mismatch error. I'm not yet sure what's the cause - it seems to be happening across all the recent CI runs, since about 3pm today.

File: tests/py/dynamo/conversion/test_bmm.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+
+from .harness import DispatchTestCase
+
+
+class TestBmmConverter(DispatchTestCase):
+    @parameterized.expand(
```
Comment (apbose) [inline tests/py/dynamo/conversion/test_bmm.py:10]: As such everything looks good. But can more test cases be added with varying dimensions?

**Review by gs-olive — CHANGES_REQUESTED** [review]
CI issues resolved - should just need the additional test cases, and then can merge

**Review by apbose — APPROVED** [review]
Looks good!

**Review by gs-olive — APPROVED** [review]
Looks good to me!
